### PR TITLE
fix: make StreamableQuery#internalStream package-private to avoid documentation and prevent clirr errors

### PR DESCRIPTION
**Pull request by MarkDuckworth**

### CODE REVIEW

File: google-cloud-firestore/src/main/java/com/google/cloud/firestore/StreamableQuery.java
Comment (tom-andersen) [inline google-cloud-firestore/src/main/java/com/google/cloud/firestore/StreamableQuery.java:1]: Could we make the entire class package private?

Comment (MarkDuckworth) [reply google-cloud-firestore/src/main/java/com/google/cloud/firestore/StreamableQuery.java:1]: It's possible that customers have declared variables of this type. `StreamableQuery query = firestore.collection(...).where(...)`. I can deprecate use of this class and remove it from the API surface in the next major release.

Comment (MarkDuckworth) [reply google-cloud-firestore/src/main/java/com/google/cloud/firestore/StreamableQuery.java:1]: Instead we decided to apply the `@InternalApi` attribute to the StreamableQuery base class.

**Review by tom-andersen — APPROVED** [review]
This aligns with the direction I want to move.

Note: I would have gone further, but in the exact same direction, so LGTM and let's get this merged.

**Pull request merged**